### PR TITLE
Handle units in some YAML settings (post CASSANDRA-15234)

### PR DESCRIPTION
**Pull request by rzvoncek**
It seems that at least due to [CASSANDRA-15234](https://issues.apache.org/jira/browse/CASSANDRA-15234) the `cassandra.yaml` file started to feature units (eg `MiB`) for some settings.

Montecristo is mostly not troubled by this, except for the `row_cache_size` where we actually parse the value to see if it's bigger than 0.

This PR improves that parsing by removing the `MiB` string. I could not find a decent(-ly simple) de-humanization library to drop in, so I went for a quick fix like this.

I'm also having issues with the unit tests. I think the `Assertions` library is not taken into the accunt. I'm commiting a failing unit test, hoping to see the CI here to fail.

### CODE REVIEW

File: montecristo/src/main/kotlin/com/datastax/montecristo/model/application/CassandraYaml.kt [hunk 1]
```diff
@@ -31,7 +31,9 @@ data class CassandraYaml(val data : JsonNode)  : YamlConfig(data) {
     val partitioner get() = get("partitioner")
     val memtableAllocationType get() = get("memtable_allocation_type")
     val authorizer get() = get("authorizer")
-    private val rowCacheSizeInMB get() = (get("row_cache_size","0")).toInt()
+    val rowCacheSizeInMB get() = (
+            get("row_cache_size", "0").replace("MiB", "")
```
Comment (michaelsembwever) [inline montecristo/src/main/kotlin/com/datastax/montecristo/model/application/CassandraYaml.kt:35]: could be GiB (or KiB), or GB, MB, KB

Comment (rzvoncek) [reply montecristo/src/main/kotlin/com/datastax/montecristo/model/application/CassandraYaml.kt:35]: Could I then delete all non-numeric characters ? Something like `.replace("\D+", "")` ?

Comment (michaelsembwever) [reply montecristo/src/main/kotlin/com/datastax/montecristo/model/application/CassandraYaml.kt:35]: yeah, i agree to that

Comment (rzvoncek) [reply montecristo/src/main/kotlin/com/datastax/montecristo/model/application/CassandraYaml.kt]: Pushed a commit doing this.

**Review by michaelsembwever — APPROVED** [review]

**Pull request merged**